benchmarks/sql-benchmark/queries_duckdb.sql

```diff
@@ -1,28 +1,34 @@
 PRAGMA enable_profiling;
 
--- Create (L2.87s)
+-- Temp directory is necessary for larger queries.
+PRAGMA temp_directory='tmp';
+
+-- Set this to the input directory to use
+-- PRAGMA file_search_path='6x_1';
```

Comment:
I *think* I added this because I was planning on making a bunch of different data sets of different sizes (eg., 6x_1, 6x_2, 6x_3, 12x_1, 12x_2, 12x_3, etc.) to measure (a) average time on a given size and (b) scaling of different systems. Hadn't finished that.